### PR TITLE
refactor: centralize the logic of choosing worker

### PR DESCRIPTION
**Pull request by QuintinTao**
I added a function in the write_worker.rs to centralize the logic of choosing worker:

pub fn find_worker(table_id: usize, worker_num: usize) -> usize {
table_id % worker_num
}

### CODE REVIEW

**Comment by archerny** [comment]
You can edit the PR title here:
image

**Comment by QuintinTao** [comment]
> * centralize the logic of choosing worker

OK

File: analytic_engine/src/instance/write_worker.rs [hunk 1]
```diff
@@ -989,7 +989,10 @@ impl WriteWorker {
         self.local.data.background_notify.notified().await;
     }
 }
-
+///Centralize the logic of choosing worker for table into one place #268
+pub fn find_worker(table_id: usize, worker_num: usize) -> usize {
+    table_id % worker_num
+}
```
Comment (chunshao90) [inline analytic_engine/src/instance/write_worker.rs:995]:
```suggestion

/// Centralize the logic of choosing worker for table into one place.
#[inline]
pub fn find_worker(table_id: usize, worker_num: usize) -> usize {
    table_id % worker_num
}

```

File: analytic_engine/src/instance/write_worker.rs [hunk 2]
```diff
@@ -989,7 +989,10 @@ impl WriteWorker {
         self.local.data.background_notify.notified().await;
     }
 }
-
+///Centralize the logic of choosing worker for table into one place #268
+pub fn find_worker(table_id: usize, worker_num: usize) -> usize {
```
[the remark this answers is not in the document]
Comment (QuintinTao) [reply analytic_engine/src/instance/write_worker.rs:993]: OK

**Review by chunshao90 — APPROVED** [review]
LGTM

**Pull request merged**